src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -826,6 +826,12 @@ export default class DefaultDeviceController
         }
       } catch (err) {
         this.logger.info('unable to get media device labels');
+        this.eventController?.publishEvent('audioInputFailed', {
```

Comment:
Just checking if firefox sends two same `audioInputFailed` when the permissions are blocked or a single one.

Reply:
I mentioned on the ticket but the reason Firefox sent previously is due to we tried to select a device with empty label so i fixed that in the demo.